### DISCUSSION TITLE
I cant go into a modded server and stay.

### DISCUSSION THREAD

**Discussion by chessbored** [FabricMC/fabric-api#2177] [Player support]
I have the same issue as this guy, except i have way more mods and im unsure of which one may be causing the issue. they have worked before, but wont now. could anyone help?
File attached are the mods I use.
[this is the guy](https://board.aternos.org/thread/80388-connection-lost-no-matter-what/) 
![Snapchat-782064052](https://user-images.githubusercontent.com/104628548/166132004-ab876bb0-834e-416b-aa0f-b2bda6036323.jpg)

**Comment by warjort** [comment]
Post your logs/latest.log for the client (your minecraft).
If you have access to it, post the logs/latest.log for the server around when it disconnects.

Most likely, there is an error in the logs that is causing the disconnect.

You can also get a disconnect if the network connection is rubbish.
Or one of the machines can't keep up with the work and the other side "times it out" because it thinks it vanished due to slow response.

I don't think your link is relevant. It includes a crash log with an error from optifine for Minecraft 1.16.5 that doesn't even look like it has fabric installed. You have fabric, minecraft 1.18.1 and no optifine.

Also, why 2 versions of fabric api in your mod folder, what is "f" and why hide some?

**Reply by chessbored** [reply]
the hidden things are just folders, and im not sure what F is. im not even sure why two fabric api are in there. someone sent the modpack to me to download.

**Reply by chessbored** [reply]
[2022-05-01-2.log](https://github.com/FabricMC/fabric/files/8598428/2022-05-01-2.log)
heres the log for it

**Comment by warjort** [comment]
First thing I see is this:
````
[02:51:59] [Render thread/INFO]: OpenGL debug message: id=1281, source=API, type=ERROR, severity=HIGH, message='GL error GL_INVALID_VALUE'
````
Which has Mojang "issue": https://bugs.mojang.com/browse/MC-148346

But there are warnings before that:
````
[02:20:26] [Render thread/WARN]: Received unknown attacker
[02:32:23] [Render thread/WARN]: Received passengers for unknown entity
[02:33:27] [Render thread/WARN]: Received passengers for unknown entity
[02:33:45] [Render thread/WARN]: Received passengers for unknown entity
[02:34:27] [Render thread/WARN]: Received passengers for unknown entity
[02:34:44] [Render thread/WARN]: Received passengers for unknown entity
[02:35:21] [Render thread/WARN]: Received passengers for unknown entity
[02:38:49] [Render thread/WARN]: Received passengers for unknown entity
[02:38:55] [Render thread/WARN]: Received passengers for unknown entity
````
These suggest you have a mismatch between mods or one of them is sending rubbish data.
You will need to contact whoever sent you the modpack or operates the server to confirm you have correct mods.
None of these are actual errors though.

Your log shows "fabric 0.46.6+1.18" which is neither of the 2 fabric api versions you showed before?

**Reply by chessbored** [reply]
ive comtacted the people who sent me it and they say that its all correct. and maybe if i fix the fabric api it may work.. its worked before so i dont understand why its doing this now

**Reply by warjort** [reply]
try reinstalling your modpack from a fresh install, the multiple fabric apis suggest you have installed over something else (an older version?) which has left stuff there that shouldn't be.

**Reply by chessbored** [reply]
i have downloaded the fresh files for it and it still wont work...

**Reply by warjort** [reply]
did you also look at the opengl error?
https://bugs.mojang.com/browse/MC-148346
Suggested fixes for that include:
* check the drivers for your graphics card are up-to-date
* disable in game overlays like steam, overwolf or discord
* you have a custom shader that is broken

**Reply by warjort** [reply]
You also want to ask if there is an error on the server when you disconnected

**Reply by chessbored** [reply]
now that ive done all that, i cant even join the server. its only putting me to timed out

**Reply by chessbored** [reply]
now back to the same connection lost

**Reply by warjort** [reply]
describe what you did/changed and post the log
we can't help with "it does not work" type problem descriptions

the symptoms you are describing (not being able to join and random disconnects) sound like a broken network
but I would expect to see an error confirming this in either your log or the server log
